run

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env sh
 
+SCRIPT_PATH=$(readlink -f "$0")
+PROJECT_DIR=$(dirname "$SCRIPT_PATH")
+cd $PROJECT_DIR
```

Comment:
> Also weder im Skript noch im Compiler selbst das Verzeichnis wechseln

If you change the directory, the argument passed to --echo will be relative to the project dir instead of the current working directory. Instead simply call the `minijavac` bash script with its absolute path (I should have done that in the first place, my bad).

`"$PROJECT_DIR/build_dir/install/minijavac/bin/minijavac" "$@"`